core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,16 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput' &&
```

Comment:
Oh, I just realized the issue, sorry! In the answer groups for the multiple-choice interaction, we currently record the index of the answer, rather than the answer itself. I forgot we do that.

So, what you'd want to do instead is similar, but instead you would look at the length of getAnswerChoices() -- say it's 3, then the full range of "answers" is [0, 1, 2]. You then want to check that each "answer" is contained within at least one answer group. To do this, in each answer group, look at the value of the rule_specs property -- for each element in this list, look at ruleSpec.inputs.x. That will give you the index of the choice that the corresponding rule is referring to. (Note that an answer group may contain more than one rule.)

Does this help?